leap/leap_test.rb

```diff
@@ -15,26 +17,38 @@ def leap?
 
 class YearTest < Minitest::Test
   def test_leap_year
-    assert Year.leap?(1996), 'Yes, 1996 is a leap year'
+    assert Year.leap?(1996)
```

Comment:
Good point, sorry I missed that. Again, I think it requires a design decision. The existing pattern does not have the messages in either the JSON file or the test generator template. While one could take either approach, I think adding the messages to the JSON file would provide consistency across languages and push less logic into the generators.
Once we have a call made, then I'd need to back-fill on the existing Ruby test generators and JSON files.
Trying the Go ones would be very dangerous:)